lib/public/components/Filters/common/filters/checkboxFilter.js

```diff
@@ -23,6 +23,7 @@ import { h } from '/js/src/index.js';
  * @param {function} onChange the handler called once the checkbox state changes (change event is passed as first parameter, value as second)
  * @param {Object} [additionalProperties] Additional options that can be given to the class.
  * @returns {vnode} An object that has one or multiple checkboxes.
+ * @deprecated
```

Comment:
Why not using this instead of deprecating it?
You should always try to propose alternative when putting deprecated, or at least a reason of deprecation

Reply:
put alternative info